app/jobs/alchemy/pg_search/index_page_job.rb

```diff
@@ -0,0 +1,10 @@
+module Alchemy
+  module PgSearch
+    class IndexPageJob < BaseJob
+      def perform(page)
+        Search::index_page page
```

Comment:
Geht so zwar auch, ist aber eher ungewöhnlicher Ruby Syntax

```suggestion
        Search.index_page(page)
```